language.md

```diff
@@ -48,7 +48,7 @@ Falls Sie Deutsch sprechen, beginnen Sie [hier](german/ruderboot.md).
 
 Euskalduna bazara, joan [hona](basque/gaztelua.md).
 
-เรียนเชิญเข้าภาษาไทย Welcome to [Thailand](Thai/Songs/ThaiSong_FanJa.md) 

```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.